oauth2-useragent-javafx/src/main/java/com/microsoft/alm/oauth2/useragent/InterceptingBrowser.java

```diff
@@ -113,6 +115,10 @@ static boolean matchesRedirection(final String expectedRedirectUriString, final
             }
         }
 
+        if (actualUri.getScheme().equals("urn") && !actualUriString.startsWith(expectedRedirectUriString)) {

```

Comment:
> So even if we use the other constructor to create a URL (instead of URI), we still have to do a string comparison

It seems fine to leave it for now.